.github/workflows/ci.yml

```
@@ -475,6 +475,7 @@ jobs:
       - run: python -m pytest modin/test/interchange/dataframe_protocol/test_general.py
       - run: python -m pytest modin/test/interchange/dataframe_protocol/hdk
       - run: python -m pytest modin/experimental/sql/test/test_sql.py
+      - run: python -m pytest -n 2 modin/pandas/test/test_concat.py
```

Not sure if this is needed for hdk
```suggestion
      - run: python -m pytest modin/pandas/test/test_concat.py
```